app/controllers/provider/admin/messages/outbox_controller.rb

```diff
@@ -20,12 +19,14 @@ def destroy
   end
 
   def create
-    @message = messages.build(message_params)
+    @message = build_message(message_params)
     if @message.valid?
       enqueue_message_and_respond
     else
-      flash[:error] = @message.errors.full_messages.to_sentence
```

Comment:
Is the flash message not used anymore?

Reply:
Not anymore, now the errors are rendered via JS as inline errors for each input (in https://github.com/3scale/porta/pull/3531/files#diff-d0c02a70007d734a61efaae1acb175ce7890d010bed1d37d1524f37838edfc1f)